spec/fixtures/responses/whois.tld.ee/ee/property_status_serverhold.expected

```diff
@@ -0,0 +1,116 @@
+#domain
+  %s == "samanacrafts.ee"
+
+#status
+  %s == :expired
+
+#available?
+  %s == false
+
+#registered?
+  %s == true
+
+#created_on
+  %s %CLASS{time}
+  %s %TIME{2014-11-01 20:38:55 +02:00}
+
+#updated_on
+  %s %CLASS{time}
+  %s %TIME{2014-11-01 20:38:55 +02:00}
+
+#expires_on
+  %s %CLASS{time}
+  %s %TIME{2015-11-01}
+
+#disclaimer
```

Comment:
Please move the disclaimer at the top. The order of the elements in the expectation file are relevant.